backend/modules/instruments/schemas.py

```
@@ -0,0 +1,106 @@
+    trailing_pe: Decimal | None = None
+    forward_pe: Decimal | None = None
+    dividend_yield: Decimal | None = None
+    earnings_date: datetime | None = None
+    business_summary: str | None = None
+    financial_data: dict | None = None
+    major_holders: dict | None = None
+    institutional_holders: list[dict] | None = None
+    analyst_recommendations: dict | None = None
+
+
+class NewsThumbnailResolution(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)